core/src/main/resources/hudson/model/AllView/noJob_de.properties

```diff
@@ -20,7 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-Welcome\ to\ Jenkins\!=Willkommen bei Jenkins!
+Welcome\ to\ Jenkins!=Willkommen bei Jenkins!
```

Comment:
that was proposed by translation-tool. also the neighbor files have the same key